framework/meta.yaml

```
@@ -11,7 +11,8 @@ source:
 
 requirements:
   build:
-    - boost {{ boost }}
+    - _openmp_mutex {{ _openmp_mutex }}
```

```suggestion
    - _openmp_mutex {{ _openmp_mutex }} [linux64]
```
This is only a linux runtime dependency I believe

---

Good call, thanks! Done!